docs/fides/docs/ui/datastores.md

```diff
@@ -22,9 +22,46 @@ All currently configured datastores will appear in the paginated Datastore Conne
 | Testing Status | Filter stores by the result of their last test: Passed, Failed, or Untested. |
 | Status | Filter datastores by status: Active or Disabled. |
 
-## Testing datastores
+## Add a datastore
 
-Each configured datastore includes an option to `Test` its connection. Fidesops will record the last tested time to the datastore's card, and update the current connection status.
+To add a new datastore connection, select "**Create New Connector**" from the Datastore Connections panel. You will be directed to a list of all available connection options, including adding [manual connectors](#manual-connections).
+
+![new connection](../img/admin_ui/new_connection.png)
+
+Search for and select your desired connector. The Datastores UI will assist in adding and configuring your new connection. 
+
+### Automated connections
+
+Fides automatically includes your third-party connections when processing [privacy requests](../getting-started/privacy_requests.md). Once you have selected a connection type, the UI allows you to describe your connection's configuration information, which includes any necessary fields for accessing and updating third-party data.
```

Comment:
It'd be good to clarify "third-party connections" here, I know elsewhere they're referred to as SaaS connectors for instance. I prefer this naming.